Makefile

```
@@ -11,7 +11,17 @@ lint::
 	golangci-lint run --timeout 5m
 
 test_acceptance::
-	go test -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM} ./tests/...
+	go test -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM} ./tests/terraform/...
+
+generate_tf2pulumi_coverage_input::
+	(cd tests/coverage-report/testdata && if [ ! -d terraform-provider-aws ]; then git clone https://github.com/hashicorp/terraform-provider-aws && cd terraform-provider-aws && git checkout 59d66d6283496aa47e90ec78d0eb3851e0a640e1; fi)
```

You don't have to provide the full hash value; just enough to disambiguate commits. 7 characters is plenty for that.
```suggestion
	(cd tests/coverage-report/testdata && if [ ! -d terraform-provider-aws ]; then git clone https://github.com/hashicorp/terraform-provider-aws && cd terraform-provider-aws && git checkout 59d66d6; fi)
```